modular_zubbers/code/modules/mob/living/silicon/robot/robot_defines.dm

```diff
@@ -0,0 +1,24 @@
+/mob/living/silicon/robot/model/centcom
+	icon = 'modular_zubbers/icons/mob/silicon/widerobot_cc.dmi'
+	icon_state = "valecc"
+	faction = list(ROLE_DEATHSQUAD)
+	req_access = list(ACCESS_CENT_GENERAL)
+	lawupdate = FALSE
+	scrambledcodes = TRUE // These are not station borgs.
+	ionpulse = TRUE
+	var/playstyle_string = "<span class='big bold'>You are a Central Command cyborg!</span><br>"
```

Comment:
there is a macro for spans

Reply:
I just took the original code, what would you suggest I place here instead?